File: 프로그래머스/1/12969. 직사각형 별찍기/직사각형 별찍기.java
import java.util.Scanner;

class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        
        int max = 0;
        max += a < b ? b : a;

        String star = "";
        for (int x=0; x < max; x++) {
            star += "*";
        }

        for (int i=0; i < b; i++) {
            System.out.println(star.substring(0, a));
        }
    }
}

File: 프로그래머스/1/12969. 직사각형 별찍기/직사각형 별찍기.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val sb = StringBuilder()
    val str = sc.nextLine()
    val list = str.split(" ")
    (0 until list[1].toInt()).forEach { 
        (0 until list[0].toInt()).forEach { i -> sb.append("*") }
        sb.append("\n")
    }
    
    println(sb.toString())
}

